tests/qat/test_instructions.py

```diff
@@ -318,6 +319,29 @@ def test_deviceinjector_reverts(self):
             execute_instructions(EchoEngine(hw), builder)
         assert measure_channel.frequency == 9e9
 
+    def test_sweep_acquire_time(self):
+        hw = get_default_echo_hardware(2)
+        qubit = hw.get_qubit(0)
+        measure_channel = qubit.get_measure_channel()
+        acquire_channel = qubit.get_acquire_channel()
+        acquire_times = [1.0e-6, 2.0e-6, 3.0e-6]
```

Comment:
If I understand correctly, the acquire times in the `Variable` can still be <0?

Reply:
EDIT: After a discussion with @keriksson-rosenqvist , we decided to ignore the comment above as it will cause a lot of breaks in dependent packages. 

We will consider this comment in the HWModel refactor.